src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -683,6 +691,8 @@ def save(self, new_comment=None, notify=True):
       self._set_severity(severity)
       self.labels.remove_by_prefix(_SEVERITY_LABEL_PREFIX)
 
+      # Check for an access_limit label before filtering
+      limit_view_label = _extract_label(self.labels, 'LIMIT_')
```

Comment:
Should we combine `_extract_label` and `_get_access_limit_for_label` into a single function? i.e. `_extract_access_limit_from_labels(self.labels)` ?

This consolidates the logic a bit.

Reply:
Done.